app/puppet-code/deploy_test.go

```diff
@@ -0,0 +1,139 @@
+package app
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/puppetlabs/pe-sdk-go/json"
+
+	"github.com/puppetlabs/pe-sdk-go/app/puppet-code/api/client"
+	"github.com/puppetlabs/pe-sdk-go/app/puppet-code/api/client/operations"
+	mock_operations "github.com/puppetlabs/pe-sdk-go/app/puppet-code/api/client/operations/testing"
+	"github.com/puppetlabs/pe-sdk-go/app/puppet-code/api/models"
+	mock_api "github.com/puppetlabs/pe-sdk-go/app/puppet-code/api/testing"
+	match "github.com/puppetlabs/pe-sdk-go/app/puppet-code/testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRunDeployFailsIfNoClient(t *testing.T) {
+	assert := assert.New(t)
+	errorMessage := "No client"
+
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	//token := mock_token.NewMockToken(ctrl)
```

Comment:
this unuseful comments should be removed (maybe also token mock, if not used anymore)